dashboard/common/db_util.py

```diff
@@ -1,13 +1,12 @@
 # Some utility functions used by other classes in this project
 from datetime import datetime
 import logging
-from typing import Dict, List, Union
-
 from dateutil.parser import parse
 import django
-from django.conf import settings
 from django_cron.models import CronJobLog
-
+from typing import Dict, List, Union
+from django.conf import settings
+import re
```

Comment:
Nit-pick: Any particular reason you are moving these around? I tried to organize them recently by standard library, third-party library, and local library, but it looks like you changed them again. Not super important in the end...

Reply:
Probably merging got it dis ordered

Reply:
Done

Reply:
Yeah this is the PEP-8 order, this utility isort will do it automatically. Would probably be nice to have a git-hook for this and a PR to just fix up all the files. 

https://github.com/timothycrosley/isort/

Reply:
Interesting a library to sort the import, I guess some people get really fuzzy about all these things

Reply:
Yeah there are a few open source projects I've contributed to that have checks which which fail if the import order isn't "correct".